go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kubeapps/kubeapps
 
-go 1.13
+go 1.16
 
 replace github.com/docker/docker => github.com/docker/docker v0.0.0-20190731150326-928381b2215c
```

Comment:
I'm wondering if it is still necessary, I'd rather say it's not, but I wanted to check it with you first

Reply:
If what's necessary? Changing the go version in go.mod, or replacing docker/docker ?

Reply:
Oops, I thought I selected the proper line, but this excerpt is confusing indeed. I meant the version replacement, I (want to) suppose that whichever change required in this version is now upstream, no?